### PR TITLE
docs: 📝 readme >>>/... removed

### PR DESCRIPTION
**Pull request by onuralpszr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,17 +46,17 @@ Read more about desktop, headless, and local installation in our [guide](https:/
...
->>> result = model(image)[0]
->>> detections = sv.Detections.from_ultralytics(result)
+image = cv2.imread(...)
+model = YOLO('yolov8s.pt')
+result = model(image)[0]
+detections = sv.Detections.from_ultralytics(result)
 
->>> len(detections)
-5
+print(len(detections))
```
Comment (SkalskiP) [inline README.md:58]: we can probably drop `print`

File: README.md [hunk 2]
```diff
@@ -67,17 +67,17 @@ Supervision was designed to be model agnostic. Just plug in any classification,
...
-    >>> result = model.infer(image)[0]
-    >>> detections = sv.Detections.from_inference(result)
+    image = cv2.imread(...)
+    model = get_roboflow_model(model_id="yolov8s-640", api_key=<ROBOFLOW API KEY>)
+    result = model.infer(image)[0]
+    detections = sv.Detections.from_inference(result)
 
-    >>> len(detections)
-    >>> 5
+    print(len(detections))
```
Comment (SkalskiP) [inline README.md:79]: we can probably drop `print`

File: README.md [hunk 3]
```diff
@@ -108,19 +108,19 @@ https://github.com/roboflow/supervision/assets/26109316/691e219c-0565-4403-9218-
...
+    data_yaml_path=...
+)
 
->>> dataset.classes
+dataset.classes
 ['dog', 'person']
 
->>> len(dataset)
-1000
+print(len(dataset))
```
Comment (SkalskiP) [inline README.md:122]: we can probably drop `print`

File: README.md [hunk 4]
```diff
@@ -129,86 +129,86 @@ Supervision provides a set of [utils](https://supervision.roboflow.com/datasets/
...
 
   ```python
-  >>> train_dataset, test_dataset = dataset.split(split_ratio=0.7)
-  >>> test_dataset, valid_dataset = test_dataset.split(split_ratio=0.5)
+  train_dataset, test_dataset = dataset.split(split_ratio=0.7)
+  test_dataset, valid_dataset = test_dataset.split(split_ratio=0.5)
 
-  >>> len(train_dataset), len(test_dataset), len(valid_dataset)
-  (700, 150, 150)
+  print(len(train_dataset), len(test_dataset), len(valid_dataset))
```
Comment (SkalskiP) [inline README.md:155]: we can probably drop `print`

**Review by SkalskiP — CHANGES_REQUESTED** [review]

**Review by onuralpszr — COMMENTED** [review]
All done

**Comment by SkalskiP** [comment]
Awesome @onuralpszr! Thank you! 🙏🏻

**Pull request merged**